litani

```diff
@@ -144,6 +145,11 @@ def get_add_job_args():
             "default": 10,
             "type": int,
             "help": "seconds between memory profile polls"
+        }, {
+            "flags": ["--phony-outputs"],
+            "metavar": "OUT",
+            "nargs": "*",
```

Comment:
Should this be changes to `+` with the added warning to use a `touchfiles` directory for dependency ordering if one chooses to not use the `--phony-outputs` parameter?